### ISSUE TITLE
Possible Unhandled Promise Rejection (id: 0)

### ISSUE THREAD

**Issue by adamk77** [mattermost/mattermost-mobile#5260]
#### Summary
After logging into the Mattermost server, the iOS app displays at the bottom of the screen: "Possible Unhandled Promise Rejection (id: 0)"

#### Environment Information

- Device Name: iPhone 12
- OS Version: 14.4.1
- Mattermost App Version:  master branch
- Mattermost Server Version: 5.33.2 Linux

#### Steps to reproduce
Compile and launch app then login

**Comment by amyblais** [comment]
cc @enahum @migbot

**Comment by migbot** [comment]
@adamk77 I can't reproduce this. Could you try logging in to https://community.mattermost.com and tell us if you see the warning?

**Comment by adamk77** [comment]
Yes, I still see the warning. I've attached the screenshot. 

![PNG image](https://user-images.githubusercontent.com/1545197/113182797-51ddd280-9221-11eb-9248-cce4af6881e1.png)

**Comment by adamk77** [comment]
Here you go.

![PNG image 2](https://user-images.githubusercontent.com/1545197/113189771-8fdef480-9229-11eb-83aa-acd53a9cec07.png)
![PNG image 3](https://user-images.githubusercontent.com/1545197/113189777-92414e80-9229-11eb-9a84-1f48cacdd0d1.png)

**Comment by adamk77** [comment]
Also including the entitlements:

Mattermost.entitlements
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>aps-environment</key>
	<string>development</string>
	<key>com.apple.developer.icloud-container-identifiers</key>
	<array>
		<string>iCloud.$(CFBundleIdentifier)</string>
	</array>
	<key>com.apple.developer.icloud-services</key>
	<array>
		<string>CloudDocuments</string>
	</array>
	<key>com.apple.developer.ubiquity-container-identifiers</key>
	<array>
		<string>iCloud.$(CFBundleIdentifier)</string>
	</array>
	<key>com.apple.developer.ubiquity-kvstore-identifier</key>
	<string>$(TeamIdentifierPrefix)$(CFBundleIdentifier)</string>
	<key>com.apple.security.application-groups</key>
	<array>
		<string>group.com.ftallc.mattermost.rnbeta</string>
	</array>
	<key>keychain-access-groups</key>
	<array>
		<string>$(AppIdentifierPrefix)com.ftallc.mattermost.rnbeta</string>
	</array>
</dict>
</plist>
```

MattermostShare.entitlements
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>com.apple.security.application-groups</key>
	<array>
		<string>group.com.ftallc.mattermost.rnbeta</string>
	</array>
</dict>
</plist>
```

NotificationService.entitlements
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>com.apple.security.application-groups</key>
	<array>
		<string>group.com.ftallc.mattermost.rnbeta</string>
	</array>
</dict>
</plist>
```

**Comment by adamk77** [comment]
Here's a screenshot of it caught in the debugger:

![Screen Shot 2021-03-31 at 2 14 31 PM](https://user-images.githubusercontent.com/1545197/113191389-79d23380-922b-11eb-837a-d65249323bc8.png)

**Comment by adamk77** [comment]
Also screenshot showing content of attributes and mAttributes.  It says "group.com.mattermost.rnbeta". Is this right? I know this was set to my own personal one.

![Screen Shot 2021-03-31 at 2 23 03 PM](https://user-images.githubusercontent.com/1545197/113192462-c8340200-922c-11eb-9945-f30141c0f557.png)

**Comment by adamk77** [comment]
I apologize for the barrage of posts.  Here is a screenshot where I searched for "com.mattermost.rnbeta". I'm not sure why all the Info.plist still has the old ids when I've changed them in XCode.  After I changed all the Info.plist to the new ids, the warning has disappeared.

![Screen Shot 2021-03-31 at 2 31 49 PM](https://user-images.githubusercontent.com/1545197/113193561-1990c100-922e-11eb-87d0-f44e6b90d8df.png)
